debian/rules

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/make -f
+
+DESTDIR = $(CURDIR)/debian/tmp
+prefix = /usr
+SOURCES_DIR = $(CURDIR)/../SOURCES
+
+include /usr/share/dpkg/default.mk
+include /usr/share/dpkg/buildtools.mk
+
+%:
+	dh $@ --with  python3 --buildsystem=pybuild -D scripts
+
+override_dh_auto_configure:
+	test -f configure || ./autogen.sh
+	./configure \
+          --prefix=$(prefix) \
+	  --libdir=$(prefix)/lib/$(DEB_HOST_MULTIARCH) \
+	  --libexecdir=$(prefix)/lib/$(DEB_HOST_MULTIARCH) \
+	  --includedir=$(prefix)/include \
+          --sbindir=$(prefix)/sbin \
+          --datadir=$(prefix)/share \
+	  --mandir=$(prefix)/share/man \
+          --docdir=$(prefix)/share/doc/geopm \
+          --with-bash-completion-dir=/etc/bash_completion.d \
+          --disable-mpi \
+          --disable-openmp \
+          --disable-fortran \
+          --disable-geopmd-local \
+	  --disable-mpi
+
+override_dh_auto_test:
+	# make check
```

Comment:
?

Reply:
The test code doesn't currently build on Ubuntu 22